lib/src/Uncertainty/Algorithm/Sensitivity/JansenSensitivityAlgorithm.cxx

```diff
@@ -75,7 +75,7 @@ JansenSensitivityAlgorithm * JansenSensitivityAlgorithm::clone() const
 Sample JansenSensitivityAlgorithm::computeIndices(const Sample & sample,
     Sample & VTi) const
 {
-  const UnsignedInteger inputDimension = inputDesign_.getDimension();
+  const UnsignedInteger inputDimension = inputDescription_.getSize();
```

Comment:
? It looks more natural to me to ask an object for its dimension in order to initialize a variable named inputDimension...